dhis-2/dhis-web/dhis-web-commons-resources/src/main/webapp/dhis-web-commons/javascripts/login.js

```diff
@@ -6,17 +6,7 @@ $( document ).ready( function()
 {
     $( '#j_username' ).focus();
 
-    var checked = document.getElementById( '2fa' ).checked;
-
-    $( '#2fa' ).click( function () {
-        $( '#2fa_code' ).attr("hidden", checked);
-        $( '#2fa_code' ).attr("readonly", checked);
-        document.getElementById( '2fa' ).checked = !checked;
-
-        checked = !checked;
-    });
-
-    $( '#loginForm').bind( 'submit', function() 
```

Comment:
This change seems unrelated, and probably shouldn't be here?

Reply:
I added back the CSP rules to the login page which was missing. Since it was related to the CSP filter. This method is just not in use anymore, so I cleaned it.